Swift/Tests/ReplicatorTest.swift

```diff
@@ -1144,6 +1144,7 @@ class ReplicatorTest_Main: ReplicatorTest {
     }
     
     func testMaxRetryWaitTimeOfReplicator() {
+        timeout = 12 // already it takes 8 secs of retry, hence 12secs timeout. 
```

Comment:
I don't remember. Is this affect the other tests?

Reply:
I was under the assumption, it will be reset in setup method, but probably not. Updated the code to set the 10secs in setup method.